test/test.transporters.ts

```diff
@@ -98,4 +98,42 @@ describe('Transporters', () => {
           done();
         });
   });
+
+  it('should use the http proxy if one is configured', async () => {
+    process.env['http_proxy'] = 'http://han:solo@proxy-server:1234';
+    const transporter = new DefaultTransporter();
+    nock('http://proxy-server:1234')
+        .get('http://example.com/fake', undefined, {
+          reqheaders: {
+            'host': 'example.com',
+            'accept': /.*/g,
+            'user-agent': /google-api-nodejs-client\/.*/g,
+            'proxy-authorization': /.*/g
+          }
+        })
+        .reply(200);
+    const url = 'http://example.com/fake';
+    const result = await transporter.request({url});
+    assert.equal(result.status, 200);
+    process.env['http_proxy'] = undefined;
```

Comment:
This clobbers the original value of `process.env.http_proxy`

Reply:
Fixed!